exercises/triangle/src/test/java/TriangleTest.java

```diff
@@ -86,32 +89,37 @@ public void verySmallTrianglesAreLegal() throws Exception {
     }
 
     @Ignore
-    @Test(expected = TriangleException.class)
+    @Test
     public void trianglesWithNoSizeAreIllegal() throws Exception {
```

Comment:
I know you didn't add this, but it would be great if you could change `throws Exception` to `throws TriangleException` as it's usually best to be as specific as you can :)